setup.py

```diff
@@ -44,7 +44,7 @@
     extras_require["all"].extend(reqs)
 
 install_requires = [
-    "reana-commons[snakemake_reports]>=0.95.0a2,<0.96.0",
+    "reana-commons[snakemake]>=0.95.0a2,<0.96.0",
```

Comment:
You may want to update `requirements.txt` since it still contains reference to `snakemake-reports`:

```console
$ rg snakemake-reports
• requirements.txt
54:reana-commons[snakemake-reports]==0.95.0a3  # via reana-workflow-engine-snakemake (setup.py)
```